packages/help-center/src/components/help-center-smooch.tsx

```diff
@@ -94,9 +95,10 @@ const HelpCenterSmooch: React.FC< { enableAuth: boolean } > = ( { enableAuth } )
 				return;
```

Comment:
In my case, when receiving a message, the recent chat is not updated because `lastMessageReceivedAt` is not set, as this function returns

Reply:
What do you mean? `isHelpCenterShown` or `setLastMessageReceivedAt`?

Reply:
But this would only work when Help Center is shown, isn't it? I mean, if it's not shown, we don't need to update any badge